Sources/Remote Logging/Crash Logging/SentryClient+Shim.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SentryClient+Shim.swift
+//  AutomatticTracksiOS
+//
+//  Created by BJ Homer on 7/27/21.
+//
+
+import Foundation
+import Sentry
+
+
+@objc
+private protocol SentryClient_InternalMethods {
```

Comment:
This file is a Swift re-implementation of `SentryClient+Shim.m`. I mostly avoided rewriting code for this project, but in this case, we would have needed an entire separate module just for this one file.

Reply:
Nice – thanks!